worker/worker.py

```diff
@@ -968,6 +970,13 @@ def worker():
     print("Waiting for the heartbeat thread to finish...")
     heartbeat_thread.join(HTTP_TIMEOUT)
 
+    # To avoid flooding the server for a failing worker in a loop,
+    # always end with a little delay.
+    # The exception is when connecting to the dev server, to aid testing
+    print("Thank you for contributing, please wait as the worker finalizes!")
+    if options.host != "dfts-0.pigazzini.it":
```

Comment:
The developer will test also with his server. If this is necessary better to use
```python
if options.host == "tests.stockfishchess.org":
```